lib/src/modules/task.cc

```diff
@@ -426,9 +438,110 @@ static fs::path getCachedTaskFile(const fs::path& task_cache_dir,
     }
 }
 
+// build a spool dir for the supporting library files requested by a multifile task
+static fs::path createInstallDir(const fs::path& spool_dir, const std::set<std::string>& download_set) {
```

Comment:
I think I'd switch this to `task_cache_dir` to be consistent with what we're actually passing. Also, the comment isn't really accurate. The spool dir is already created.

Reply:
When we talk about a spool dir, that's the location results of a run are put. This is something else.

Reply:
I was using `task_cache_dir` for debugging. Still planning on using spool/results dir.

Reply:
Settled on using `request.resultDir()` with a fallback to `task_cache_dir_` when not available (Should only happen in tests when `RequestType::Blocking` request is made).